modules/base/main.tf

```diff
@@ -48,11 +48,6 @@ data "aws_caller_identity" "current" {}
 
 locals {
   default_aws_auth_users = [
-    {
```

Comment:
Why we are removing this user?

Reply:
This is handled by the `enable_cluster_creator_admin_permissions = true`, which also creates an admin for the current user.